client/src/components/Admin/ImportOrganizations/ImportFileTable.js

```diff
@@ -47,11 +22,29 @@ const flattenHours = (daysArray) => {
 
 const ImportFileTable = (props) => {
   const { tenantName, data, handleImportAction, handleCancel } = props;
-  const classes = useStyles();
 
   return (
-    <section className={classes.section}>
-      <div className={classes.tableBanner}>
+    <Box
+    sx={(theme) => ({
+    marginTop: theme.spacing(2),
+    margin: "0 auto",
```

Comment:
Hi @93Belen, I haven't been able to view the changes as well yet, will let you know if I figure it out. Just wanted to share another point which I saw in one of my issues, ```margin: "0 auto",``` might override ```marginTop: theme.spacing(2),```, so we can use ```mx: 'auto',``` instead. Rest all looks good to me. Thank you.

Reply:
Done! :)